test/e2e/regression/api_test.go

```diff
@@ -120,7 +122,7 @@ func (s *APIRegressionTestSuite) TestOrganizationAPI() {
 	s.Run("2. user attached to an org as member should have no basic permission other than membership", func() {
 		createOrgResp, err := s.testBench.Client.CreateOrganization(ctxOrgAdminAuth, &shieldv1beta1.CreateOrganizationRequest{
 			Body: &shieldv1beta1.OrganizationRequestBody{
-				Name: "org acme 2",
+				Name: "org-acme-2",
```

Comment:
Any reason we added hyphen in name instead of space?

Reply:
Will restore back to original since we have the proto file changed. Earlier name regex didn't support spaces

Reply:
done